lib/setup.js

```diff
@@ -10,23 +10,18 @@ PouchDB.preferredAdapters = [];
 
 PouchDB.prefix = '_pouch_';
 
-var eventEmitter = new EventEmitter();
-
-var eventEmitterMethods = [
-  'on',
-  'addListener',
-  'emit',
-  'listeners',
-  'once',
-  'removeAllListeners',
-  'removeListener',
-  'setMaxListeners'
-];
-
-eventEmitterMethods.forEach(function (method) {
-  PouchDB[method] = eventEmitter[method].bind(eventEmitter);
-});
-PouchDB.setMaxListeners(0);
+var eventEmitter = new EE();
+
+function bindEventEmitterMethods(Pouch) {
+  Object.keys(EE.prototype).forEach(function (key) {
+    if (typeof EE.prototype[key] === 'function') {
+      Pouch[key] = eventEmitter[key].bind(eventEmitter);
+    }
+  });
+}
+
+bindEventEmitterMethods(PouchDB);
+
```

Comment:
Explicitly listing out all the EventEmitter prototype methods is not necessary because we can determine it programmatically. Hopefully this should minify a bit smaller.